### ISSUE TITLE
Does it support postfix completion

### ISSUE THREAD

**Issue by dragove** [minad/corfu#334]
I'm trying to use postfix completion with corfu and eglot. Postfix snippets are provided by some language servers. (e.g. rust-analyzer, jdtls). When I typed something like

```rust
let x = true;
x.if
```

and at the point after `if`. I can expand it to

```rust
let x= true;
if x {
    // after expanding cursor position
}
```

I can't find these snippets in corfu with eglot. So I'm looking for help here.

**Comment by minad** [comment]
Yes, this should work. Afaik you should install yasnippet for the snippet expansion.

**Comment by dragove** [comment]
> Yes, this should work. Afaik you should install yasnippet for the snippet expansion.

I've installed tempEl, And After I typed

```rust
let my_var = true;
my_var.if 
```
And call `tempel-expand` to expand, it becomes

```rust
let my_var = true;
my_var.if | {
|
}
```

The `|` symbol is to present placeholders.
